src/Components/Home/Home.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import Header from './Header/Header.js';
+import Shop from './Shop/Shop.js';
+import OrderSummary from './OrderSummary/OrderSummary.js';
+
+import fakeData from '../../fakeData/index.js';
+
+class Home extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      searchInput : "",
+      cart : { no: 0 , itemsKey : [] }
+    }
+    this.handleInput = this.handleInput.bind(this);
+    this.handleAddToCart = this.handleAddToCart.bind(this);
+  }
+
+  handleInput(e){
+    e.preventDefault();
+    const input = e.target.value;
+    this.setState({ searchInput : input});
+  }
+
+  handleAddToCart(itemKey){
+    const no_of_cart_items = this.state.cart.no;
```

Comment:
Camel case is preferred in JavaScript based projects. [Corresponding Airbnb style guide rule](https://github.com/airbnb/javascript#naming--camelCase)